services/frontend-service/Makefile

```diff
@@ -42,6 +42,9 @@ export KUBERPULT_CDSERVER?=localhost:8443
 run: bin/main
 	./bin/main
 
+start: src/api/api.ts
```

Comment:
Should this also depend on another target, or is just having the `api.ts` file enough to run the app?

Reply:
It is enough to start the ui although you need to run the frontend and backend services for KP to run. but it doesn't depend on them. / it doesn't break. :+1: